react/src/css/App.css

```diff
@@ -352,13 +344,52 @@
 .box {
     background: var(--box-bg);
     color: var(--box-fg);
-    border-radius: 6px;
-    border: 2px solid var(--box-fg);
-    border-color: var(--box-fg);
-    padding: 0.6em 1.0em;
+    /* border-radius: 6px; */
+    border-radius: 0px 10px 10px 10px;
```

Comment:
Why does the radius vary by position?  I'm not even sure I understand what a zero radius means. But regardless, does this really seam together OK? What does it accomplish?

Reply:
This is just a UI flourish that I sort of like, visually/aesthetically. It rounds all corners except the upper left one. Actually saw this on fivethirtyeight.com (visit a lot during elections ;) ), and I just liked the look.